.scrutinizer.yml

```diff
@@ -1,17 +1,33 @@
 build:
-    environment:
-        elasticsearch: false
-        memcached: false
-        mongodb: false
-        mysql: false
-        neo4j: false
-        postgresql: false
-        rabbitmq: false
-        redis: false
-
-tools:
-    external_code_coverage: true
+    nodes:
+        analysis:
+            environment:
+                php:
+                    version: 7.3
+            cache:
+                disabled: false
+                directories:
+                    - ~/.composer/cache
+            project_setup:
+                override: true
+            tests:
+                override:
+                    - php-scrutinizer-run
+                    - phpcs-run
+    dependencies:
+        override:
+            - COMPOSER_ARGS="--ignore-platform-reqs --prefer-stable" make
 
 checks:
     php:
         code_rating: true
+
+tools:
+    external_code_coverage:
+        timeout: 3600
+        runs: 6
```

Comment:
how to calculate this number?

Reply:
... it's based on the Travis config ... ale the builds that push the results to Scrutinizer

Reply:
But why so many? 🤔 I see nine builds in total. There should probably be only two (7.2 and ExtensionTransport I guess?) running xdebug and computing coverage

![image](https://user-images.githubusercontent.com/327717/60081041-b78e5800-9739-11e9-94f1-a7d67dbc7aca.png)

Reply:
If you can say for sure that the coverage won't change between PHP and lib versions, then yeah, 2 would be enough.

Reply:
Yup, I'm used to having only one run with xdebug enabled (that runs slower than others) that computes coverage, it doesn't change between versions in most cases.